### ISSUE TITLE
Check performance of animated semantics changes on low-end devices

### ISSUE THREAD

**Issue by Hixie** [flutter/flutter#2334]
If we do an animation at 60fps, we are going to call `sendAccessibilityEvent(node.id, AccessibilityEvent.TYPE_WINDOW_CONTENT_CHANGED);` every frame to invalidate the entire tree, which might be problematic on low-end devices.

**Comment by liyuqian** [comment]
In 2020, Moto G4 should be a low-end device, so I wonder if someone could write a test that reveals this performance issue in our device lab?
